### ISSUE TITLE
Installation fails on Ubuntu 13.10

### ISSUE THREAD

**Issue by hseagle2015** [grimborg/gitgitlab#2]
Hi there. I'm having trouble installing GitGitlab on Ubuntu 13.10 with Python 2.7.5+. This is the error I'm getting during installation:

```
user@ubuntero:~$ sudo pip install GitGitlab
Downloading/unpacking GitGitlab
  Downloading GitGitlab-1.0.macosx-10.9-x86_64.tar.gz
  Running setup.py egg_info for package GitGitlab
    Traceback (most recent call last):
      File "<string>", line 14, in <module>
    IOError: [Errno 2] No such file or directory: '/home/user/build/GitGitlab/setup.py'
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 14, in <module>

IOError: [Errno 2] No such file or directory: '/home/user/build/GitGitlab/setup.py'

----------------------------------------
Command python setup.py egg_info failed with error code 1 in /home/user/build/GitGitlab
Storing complete log in /home/user/.pip/pip.log
```

The complete pip log looks like this:

```
------------------------------------------------------------
/usr/local/bin/pip run on Mon Jan 27 13:50:56 2014
Downloading/unpacking GitGitlab

  Getting page http://pypi.python.org/simple/GitGitlab
  URLs to search for versions for GitGitlab:
  * http://pypi.python.org/simple/GitGitlab/
  Getting page http://pypi.python.org/simple/GitGitlab/
  Analyzing links from page https://pypi.python.org/simple/GitGitlab/
    Found link https://pypi.python.org/packages/any/G/GitGitlab/GitGitlab-1.0.macosx-10.9-x86_64.tar.gz#md5=d6403f7de9749576a405c675bbff3e3f (from https://pypi.python.org/simple/GitGitlab/), version: 1.0.macosx-10.9-x86_64
    Found link https://pypi.python.org/packages/source/G/GitGitlab/GitGitlab-1.0.tar.gz#md5=c341764c5650b2473cb6685584ab7ceb (from https://pypi.python.org/simple/GitGitlab/), version: 1.0
  Using version 1.0.macosx-10.9-x86_64 (newest of versions: 1.0.macosx-10.9-x86_64, 1.0)
  Downloading GitGitlab-1.0.macosx-10.9-x86_64.tar.gz

  Downloading from URL https://pypi.python.org/packages/any/G/GitGitlab/GitGitlab-1.0.macosx-10.9-x86_64.tar.gz#md5=d6403f7de9749576a405c675bbff3e3f (from https://pypi.python.org/simple/GitGitlab/)
  Running setup.py egg_info for package GitGitlab

    Traceback (most recent call last):

      File "<string>", line 14, in <module>

    IOError: [Errno 2] No such file or directory: '/home/user/build/GitGitlab/setup.py'

    Complete output from command python setup.py egg_info:

    Traceback (most recent call last):

  File "<string>", line 14, in <module>

IOError: [Errno 2] No such file or directory: '/home/user/build/GitGitlab/setup.py'

----------------------------------------

Command python setup.py egg_info failed with error code 1 in /home/user/build/GitGitlab

Exception information:
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/pip-1.1-py2.7.egg/pip/basecommand.py", line 104, in main
    status = self.run(options, args)
  File "/usr/local/lib/python2.7/dist-packages/pip-1.1-py2.7.egg/pip/commands/install.py", line 245, in run
    requirement_set.prepare_files(finder, force_root_egg_info=self.bundle, bundle=self.bundle)
  File "/usr/local/lib/python2.7/dist-packages/pip-1.1-py2.7.egg/pip/req.py", line 1009, in prepare_files
    req_to_install.run_egg_info()
  File "/usr/local/lib/python2.7/dist-packages/pip-1.1-py2.7.egg/pip/req.py", line 225, in run_egg_info
    command_desc='python setup.py egg_info')
  File "/usr/local/lib/python2.7/dist-packages/pip-1.1-py2.7.egg/pip/__init__.py", line 256, in call_subprocess
    % (command_desc, proc.returncode, cwd))
InstallationError: Command python setup.py egg_info failed with error code 1 in /home/user/build/GitGitlab
```

If you need additional info, I'll gladly provide it to you.

**Comment by grimborg** [comment]
Could you try writing the package name all in lower case?

sudo pip install gitgitlab

I've tested it both system wide and in a virtualenv on Ubuntu 13.10 and it works for me.

I've also removed the binary package, which maybe was the one causing you trouble. Let me know if it works!

**Comment by hseagle2015** [comment]
Capital letters don't matter. Earlier I tried running with and without capital letters and I always got the same error.

I've just ran the same command again and GitGitlab was installed successfully. I guess the problem was related with binary package. If you take a look at the error log I posted earlier today, it says "Using version 1.0.macosx-10.9-x86_64". This wasn't the case this last time, when the package was successfully installed.

Thanks for your help and for hacking custom Gitlab server support so quickly!
